Package.swift

```diff
@@ -14,7 +14,7 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/apple/swift-docc-plugin", from: "1.0.0"),
         .package(url: "https://github.com/krzyzanowskim/STTextView", exact: "0.0.20"),
-        .package(url: "https://github.com/ChimeHQ/SwiftTreeSitter", exact: "0.6.1"),
```

Comment:
I might be wrong here but I think this was just changed to use the exact version?

Reply:
Oh thanks, this was for testing purposes only!